react/src/Endpoints.tsx

```diff
@@ -0,0 +1,304 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+type Props = {
+    accessed: boolean;
```

Comment:
is this being used?

Reply:
Yes, for some react components it looks like. Gives breaking errors if removed